CalibCalorimetry/HcalTPGAlgos/src/HcaluLUTTPGCoder.cc

```diff
@@ -596,6 +595,7 @@ void HcaluLUTTPGCoder::update(const HcalDbService& conditions) {
 
       auto tpParam = conditions.getHcalTPChannelParameter(cell, false);
       int weight = tpParam->getauxi1();
+      int factorGeVPerCount = tpParam->getauxi2();
 
       int lutId = getLUTId(cell);
       int lutId_ootpu = lutId + sizeZDC_;
```

Comment:
```suggestion
      const int lutId = getLUTId(cell);
      const int lutId_ootpu = lutId + sizeZDC_;
```

Reply:
took suggestion, resolved in latest version